DependencyInjection/CmfRoutingExtension.php

```diff
@@ -289,6 +294,21 @@ public function loadSonataPhpcrAdmin($config, XmlFileLoader $loader, ContainerBu
         $loader->load('admin-phpcr.xml');
     }
 
+    public function loadResourceProvider($config, XmlFileLoader $loader, ContainerBuilder $container)
+    {
+        $container->setParameter($this->getAlias() . '.resource.repository_name', $config['repository_name']);
+
+        if ($config['map_phpcr']) {
+            $container->setParameter($this->getAlias() . '.backend_type_phpcr', true);
+        }
+
+        if ($config['map_orm']) {
+            $container->setParameter($this->getAlias() . '.backend_type_orm', true);
+        }
```

Comment:
We have a small problem here: The resource bundle could potentially use either or both of the PHPCR/ORM Route document/entities. So I have added options to trigger the adding of those folders to doctrine.

However, we also need to have the "prefix" stuff working at least for the PHPCR-ODM documents.

Any ideas?